cli-dotnet-test.js

```diff
@@ -85,11 +143,18 @@ const dotnetTest = {
 program
     .usage("option")
     .description(dotnetTest.description())
-    .option("-s, --skip-clean", dotnetTest.descriptionSkipClean())
+    .option("--by-project", "Runs all test projects for the solution in serial")
```

Comment:
Minor, but should this read as `serially` instead of `in serial` ?

Reply:
I think you're _technically_ right since serially is an adverb which would describe _Runs_ whereas serial itself is an adjective. I can update that 😛

Reply:
One reason why I prefer formal languages 😆